test/plots/rectanglePlotTests.ts

```diff
@@ -40,6 +39,44 @@ describe("Plots", () => {
       svg.remove();
     });
 
+    it("retrieves the correct entity under a point", () => {
+      var xScale = new Plottable.Scales.Linear();
+      var yScale = new Plottable.Scales.Linear();
+      var svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+      var dataset = new Plottable.Dataset(DATA);
+      var plot = new Plottable.Plots.Rectangle()
+        .x((d) => d.x, xScale).x2((d) => d.x2)
+        .y((d) => d.y, yScale).y2((d) => d.y2);
+      plot.addDataset(dataset).renderTo(svg);
+      var entities = plot.entitiesAt({ x: 140, y: 140 });
+      assert.lengthOf(entities, 1, "a single entity is retrieved at (140, 140)");
+      assert.strictEqual(entities[0].index, 2, "entity retrieved is at index 2");
+      svg.remove();
+    });
+
+    it("retrieves correct entities under a point", () => {
+      var xScale = new Plottable.Scales.Linear();
+      var yScale = new Plottable.Scales.Linear();
+      var svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+      var dataset = new Plottable.Dataset([
+        { x: 1, y: 1, x2: 3, y2: 3 },
+        { x: 4, y: 2, x2: 2, y2: 4 }
+      ]);
+      var plot = new Plottable.Plots.Rectangle()
+        .x((d) => d.x, xScale).x2((d) => d.x2)
+        .y((d) => d.y, yScale).y2((d) => d.y2);
+      plot.addDataset(dataset).renderTo(svg);
+      var entities = plot.entitiesAt({ x: 150, y: 150 });
+      assert.lengthOf(entities, 2, "two entities are retrieved at (150, 150)");
```

Comment:
Can we check which are the 2 entities returned. Either by checking their index, or otherwise.